R/theme.R

```diff
@@ -469,6 +469,27 @@ theme <- function(...,
                   validate = TRUE) {
   elements <- find_args(..., complete = NULL, validate = NULL)
 
+  elements <- fix_theme_deprecations(elements)
```

Comment:
I've separated this in its own function as all these deprecations became somewhat distracting in ascertaining what `theme()` actually does.